inference-engine/src/preprocessing/arm_neon/ie_preprocess_gapi_kernels_neon.cpp

```diff
@@ -218,14 +218,15 @@ CV_ALWAYS_INLINE void horizontal_anyLPI(std::array<std::array<uint8_t*, 4>, chan
 template<int chanNum>
 CV_ALWAYS_INLINE void horizontal_4LPI(std::array<std::array<uint8_t*, 4>, chanNum>& dst,
                                       const uchar* tmp, const short mapsx[],
-                                      const short clone[], const int length) {
+                                      const short clone[], const uchar _mask_horizontal[],
+									  const int length) {
     constexpr int nlanes = static_cast<int>(v_uint8::nlanes);
     const int half_nlanes = nlanes / 2;
     GAPI_Assert(length >= half_nlanes);
 
     const int shift = static_cast<int>(half_nlanes / 4);
 
-    uchar _mask_horizontal[nlanes] = { 0, 4, 8, 12, 2, 6, 10, 14, 1, 5, 9, 13, 3, 7, 11, 15 };
+    //uchar _mask_horizontal[nlanes] = { 0, 4, 8, 12, 2, 6, 10, 14, 1, 5, 9, 13, 3, 7, 11, 15 };
```

Comment:
Can we remove commented code?

Reply:
As you can see it's draft. I'll request a review as soon as this PR be ready for review.